### PR TITLE
[CAS-916] Add Client Setup docs

### PR DESCRIPTION
**Pull request by adasiewiczr**
https://stream-io.atlassian.net/browse/CAS-916

### 🎯 Goal

Move Client Setup docs

### 🎨 UI Changes
Screenshot 2021-04-27 at 14 01 14

### 🧪 Testing

1. Go to `documentation` folder
2. Run `yarn run start`
3. Go to Client -> Setup section

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (CMS, cookbooks, tutorial)
- [x] Reviewers added

### CODE REVIEW

**Comment by leandroBorgesFerreira** [comment]
Important: To install docusaurus also run: `yarn install`

**Review by leandroBorgesFerreira — APPROVED** [review]
Looks good to me. 🚢 🚢 🚢 🚢

**Comment by leandroBorgesFerreira** [comment]
One feedback: The **Logging** section looks too granular to me. I don't see the need to separate **Enabling Logs**, **Intercepting Logs** and **Finding Logs** in to sections. We could put everything in the same file inside "Logging" as each section is about 2 lines of comments and some code. 

Just an option, feel free to do it the way you prefer.

File: documentation/docs/client/setup/logging/enabling_logs.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (leandroBorgesFerreira) [inline documentation/docs/client/setup/logging/enabling_logs.md:1]: The path of this file should be ` documentation/docs/client/logging/enabling_logs.md`. **Logging** doesn't belong to **Setup**

**Review by leandroBorgesFerreira — CHANGES_REQUESTED** [review]
Just saw a small problem with the category of **Logging**

File: documentation/docs/client/setup/logging/enabling_logs.md (hunk 1 above)
Comment (leandroBorgesFerreira) [reply documentation/docs/client/setup/logging/enabling_logs.md:1]: The position should be 28 as this is the 28th topic of the cookbook

Comment (adasiewiczr) [reply documentation/docs/client/setup/logging/enabling_logs.md:1]: @leandroBorgesFerreira we don't follow the structure from Cookbook. We have a separate doc that describes sections and positions

**Review by leandroBorgesFerreira — APPROVED** [review]

**Pull request merged**